udata_search_service/infrastructure/migrate.py

```diff
@@ -27,11 +28,12 @@ def set_alias(index_suffix_name: str, indices: List[str] = None, delete: bool =
     es = Elasticsearch([{'host': ELASTIC_HOST, 'port': ELASTIC_PORT}])
 
     indices = indices or ALL_INDICES
-    for index_alias in indices:
-        if index_alias not in ALL_INDICES:
-            logging.error('Unknown index alias %s', index_alias)
+    for index in indices:
+        if index not in ALL_INDICES:
```

Comment:
```suggestion
    if any([x for x in indices if x not in ALL_INDICES]):
```

Reply:
I need to print the index not in ALL_INDICES though.

Maybe this instead?
```
    if any(index not in ALL_INDICES for index in indices):
        logging.error('Unknown index alias %s', next(x for x in indices if x not in ALL_INDICES))
```

Reply:
Oups did not see that. Current solution is fine.

Reply:
Or use walrus operator if supported 😇